scripts/utils.js

```diff
@@ -35,6 +36,12 @@ async function initGitConfig(repository) {
   await executeCommand(`git remote set-url origin ${repository}`)
 }
 
+function readCiVariable(variableName) {
+  const regexp = new RegExp(`${variableName}: (.*)`)
+  const ciFileContent = fs.readFileSync(CI_FILE, { encoding: 'utf-8' })
+  return regexp.exec(ciFileContent)?.[1]
+}
+
 async function replaceCiVariable(variableName, value) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** `‏readingCiFileVariable`/`replaceCiFileVariable` could be clearer